Cargo.toml

```diff
@@ -81,6 +81,7 @@ backend_drm = ["drm", "drm-ffi"]
 backend_gbm = ["gbm", "cc", "pkg-config"]
 backend_gbm_has_fd_for_plane = []
 backend_gbm_has_create_with_modifiers2 = []
+backend_gbm_has_bo_write = []
```

Comment:
One other thought I've had for these autodetected is an explicit disable. That makes testing breakages easier. TBH though, I don't know of a good way to do that.